app/spec/factories/pinwheel_account.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :pinwheel_account, class: "PinwheelAccount" do
+    cbv_flow_id { "123456" }
```

Comment:
Since PinwheelAccounts are associated to a CbvFlow, you can just put `cbv_flow` here and FactoryBot will automatically use the `cbv_flow` factory to create a CbvFlow whenever creating the pinwheel account in test.

```suggestion
    cbv_flow
```

It's one of the nice features of FactoryBot - that it will create dependent resources for you automatically by default so your models are actually valid and represent real data.